modules/library/src/main/scala/zio/elasticsearch/result/ElasticResult.scala

```diff
@@ -65,6 +67,8 @@ final class SearchResult private[elasticsearch] (private val hits: List[Item]) e
     }
 
   def items: UIO[List[Item]] = ZIO.succeed(hits)
+
+  def lastSortValue(): ZIO[Any, Nothing, Option[Json]] = ZIO.succeed(lastSort)
```

Comment:
Let's use `UIO` here. Also, it can be `lazy val`, right? Also, `items` can be `lazy val`.